src/Lecture2.hs

```diff
@@ -97,7 +110,9 @@ spaces.
 
 🕯 HINT: look into Data.Char and Prelude modules for functions you may use.
 -}
-dropSpaces = error "TODO"
+
+dropSpaces :: String -> String
+dropSpaces = head . words
```

Comment:
That's a clever solution that relies on Haskell laziness 😅 

Alternatively, you can implement this function as:

```haskell
dropSpaces = takeWhile (not . isSpace) . dropWhile isSpace
```